h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -876,13 +874,10 @@ void defaultGBMs(Key<Grid> gridKey) {
     glmParameters._family = getResponseColumn().isBinary() && !(getResponseColumn().isNumeric()) ? GLMModel.GLMParameters.Family.binomial :
             getResponseColumn().isCategorical() ? GLMModel.GLMParameters.Family.multinomial :
                     GLMModel.GLMParameters.Family.gaussian;  // TODO: other continuous distributions!
-
+    glmParameters._missing_values_handling = DeepLearningModel.DeepLearningParameters.MissingValuesHandling.MeanImputation;
     Map<String, Object[]> searchParams = new HashMap<>();
-    glmParameters._alpha = new double[] {0.0, 0.2, 0.4, 0.6, 0.8, 1.0};  // Note: standard GLM parameter is an array; don't use searchParams!
-    // NOTE: removed MissingValuesHandling.Skip for now because it's crashing.  See https://0xdata.atlassian.net/browse/PUBDEV-4974
-    searchParams.put("_missing_values_handling", new DeepLearningModel.DeepLearningParameters.MissingValuesHandling[] {DeepLearningModel.DeepLearningParameters.MissingValuesHandling.MeanImputation /* , DeepLearningModel.DeepLearningParameters.MissingValuesHandling.Skip */});
-
-    Job<Grid>glmJob = hyperparameterSearch("GLM", glmParameters, searchParams);
+    searchParams.put("_alpha", new Double[][] {{0.0}, {0.2}, {0.4}, {0.6}, {0.8}, {1.0}});
```

Comment:
Hmm, why you cannot pass params directly as `_alpha  = new double {0.0, 0.2, 0.4 ..., 1.0}`and let GLM to to grid search internally?

Reply:
In this case I want the 6 GLM's. If I pass into GLM itself then the return will be a single GLM, which is not what I want. Also, an additional aspect to this PR is another stacked ensemble that does not include GLMs, which would be more clear if we have a variety of GLMs to compare.